src/state/globalState.ts

```diff
@@ -13,6 +13,7 @@ export function createGlobalState(): Atom<GlobalState> {
     const defaultState: GlobalState = {
         modules: [],
         panels: [],
+        ui: {},
```

Comment:
panelUi? Eller er det ui du kan bruke hvor som helst?

Reply:
Eller panelStyling, eller panelCss? hvis du går for CssProperties som type

Reply:
panelUI er jeg enig i ;)

Reply:
Her ble det ui, da den nå er gjort generisk og kan brukes til mye mer